xwiki-rendering-macros/xwiki-rendering-macro-figure/src/test/resources/macrofigure5.test

```diff
@@ -0,0 +1,49 @@
+.runTransformations:macro
+.#-----------------------------------------------------
+.input|xwiki/2.0
```

Comment:
please add here a description of this test does. Thx

Reply:
I have added a description in 4be50bc1db612ed4699d73267188a21895f28675.